HelloWorld/app/src/androidTest/java/net/boondockradio/helloworld/ApplicationTest.java

```diff
@@ -0,0 +1,14 @@
+package net.boondockradio.helloworld;
+
+import android.app.Application;
+import android.test.ApplicationTestCase;
+
+/**
+ * <a href="http://d.android.com/tools/testing/testing_android.html">Testing Fundamentals</a>
+ */
+public class ApplicationTest extends ApplicationTestCase<Application> {
```

Comment:
In android we can run 2 different types of tests.

instrumentation tests and unit tests.

`androidTest` package = instrumentation tests
`test` package = unit test